source/includes/_wpc.md

```diff
@@ -253,6 +254,7 @@ curl -H "Authorization: Bearer OAUTH_ACCESS_TOKEN" \
 | end_date | String | WPC End Date |
 | channel | String | Channel of WPC |
 | geo_type | String | Geo Type of WPC |
+| display_name | String | metadata value of offer |
```

Comment:
You and I know it's the metadata of the offer, but an external caller probably won't know or care what an offer is. And if they do, I don't want to get into a debate with them about how the actual display names comes from somewhere else.    Just say "The display name of the campaign".  It adds nothing that isn't in the field name already but it's safe.